src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -12,8 +13,78 @@ import computeStateName from "../../../common/entity/compute_state_name";
 export default dedupingMixin(
   (superClass) =>
     class extends NavigateMixin(EventsMixin(superClass)) {
-      handleClick(hass, config) {
-        const tapAction = config.tap_action || "more-info";
+      registerMouse(config) {
+        let ripple = null;
+
+        const createRipple = () => {
+          if (ripple === null) {
+            ripple = document.createElement("mwc-ripple");
+            this.shadowRoot.querySelector("div").appendChild(ripple);
+            ripple.setAttribute("unbounded", "");
+          }
+        };
+
+        const startAnimation = () => {
+          if (ripple) {
```

Comment:
Don't we have a guaranteed ripple when we come here?

Reply:
Also wouldn't it make more sense to just embed createRipple inside this function? No need to do it earlier than that we  want to show it. _Do less and be lazy._

Reply:
True.

It seems the ripple needs some time to render or something before it can be activated. Perhaps it resolves an `updateComplete` or something that can be used...